pkg/tests/apis/types.go

```diff
@@ -24,22 +19,3 @@ type AnyResourceList struct {
 
 	Items []map[string]any `json:"items,omitempty"`
 }
-
-// Read local JSON or YAML file into a resource
-func (c *K8sTestHelper) LoadAnyResource(fpath string) AnyResource {
-	c.t.Helper()
-
-	//nolint:gosec
-	raw, err := os.ReadFile(fpath)
-	require.NoError(c.t, err)
-	require.NotEmpty(c.t, raw)
-
-	res := &AnyResource{}
-	if json.Valid(raw) {
-		err = json.Unmarshal(raw, res)
-	} else {
-		err = yaml.Unmarshal(raw, res)
```

Comment:
replaced with https://github.com/grafana/grafana/pull/77245/files#diff-dccf503a3a5cb24235814bc0cb6398cb9a31ba40402dd101269e854f519ef8f6R304 -- that uses the same support built into client-go